### PR TITLE
Feature/home screen

### PR DESCRIPTION
**Pull request by ioanagirbap33**

### CODE REVIEW

File: MarketMatrix/src/components/HomeScreenComponents/StoryModal.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+          <View style={styles.modalContent}>
+            <View style={styles.modalHeader}>
+              <Image
+                source={logo}
+                style={{width: 50, height: 50, alignSelf: 'center'}}
+              />
+              <Text style={styles.title}>{title}</Text>
+            </View>
+
+            {/* <View
```
Comment (roxanap33) [inline MarketMatrix/src/components/HomeScreenComponents/StoryModal.tsx:55]: Is this necessary?

Comment (adrianacostep33) [inline MarketMatrix/src/components/HomeScreenComponents/StoryModal.tsx:55]: Delete this if you don't need

File: MarketMatrix/src/navigation/MainNavigation.tsx [hunk 2]
```diff
@@ -4,12 +4,12 @@ import AuthStack from './AuthStack';
 import AuthenticatedStack from './AuthenticatedStack';
 
 const MainNavigation = () => {
-  const {isAuthenticated} = useAuth();
-  if (isAuthenticated) {
-    return <AuthenticatedStack />;
-  } else if (!isAuthenticated) {
-    return <AuthStack />;
-  }
+  // const {isAuthenticated} = useAuth();
```
Comment (adrianacostep33) [inline MarketMatrix/src/navigation/MainNavigation.tsx:7]: And those

Comment (roxanap33) [inline MarketMatrix/src/navigation/MainNavigation.tsx:7]: Leave these uncommented

**Review by adrianacostep33 — APPROVED** [review]

**Review by roxanap33 — APPROVED** [review]

**Pull request merged**